### PR TITLE
docs: release notes draft

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description
+ Adding a first version of the release notes
+ backported changelog notes related to v1.1 release

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- `x/leverage` module, allowing anyone to:
+  - supply liquidity and earn rewards,
+  - collaterize the supplied liquidity to be able to borrow,
+  - borrow (will charge the user borrow interest),
+  - any umee holder governs the `x/leverage` parameters like: token registry and borrow / supply settings. See more about the parameters in [leverage.proto](https://github.com/umee-network/umee/blob/main/proto/umee/leverage/v1/leverage.proto) file.
+- `x/oracle` module - a decentralized price oracle for the `x/leverage` module, as well as any app built in the Umee blockchain. In the future version we will offer oracle queries through IBC. Any Umee holders will govern oracle parameters defined in [oracle.proto](https://github.com/umee-network/umee/blob/main/proto/umee/oracle/v1/oracle.proto) file.
+
+#### x/leverage settings
+
+- TODO: important information about queries, and node parameters, eg: https://github.com/umee-network/umee/issues/1400
```
Comment (robert-zaremba) [inline RELEASE_NOTES.md:22]: @toteki -- do you want to add the instructions here or you prefer to do it in a separate PR?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Gravity Bridge
+
+In `v1.1.x` (existing mainnet) we disable Gravity Bridge (GB) module due to Ethereum PoS migration (_the merge_).
+This release is the first step to enable GB. We start by enabling validators update end evidence messages (`ValsetConfirm`), but the bridge messages: batch creation, claims (both ways: Ethereum->Cosmos and Cosmos->Ethereum) remain disabled.
+
+See (TODO:) Gravity Bridge Release Notes.
```
Comment (robert-zaremba) [inline RELEASE_NOTES.md:29]: we can merge this PR now, or later once I will update GB

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+### Update notes
+
+Each validator must run:
+
+- Peggo (Gravity Bridge Orchestrator).
+- Price Feeder.
+
+Instructions: https://umeeversity.umee.cc/validators/mainnet-validator.html
```
Comment (robert-zaremba) [inline RELEASE_NOTES.md:38]: @khoerling LMK if you would like to add something (or feel free to push to the branch).

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+<!-- markdownlint-disable MD013 -->
+<!-- markdownlint-disable MD024 -->
+
+# Release Notes
+
+## v3.0.0
+
+v3.0.0 follows the _umeemania_ testnet release (v2.0.x) which presented our **lending** and **oracle** functionality.
+Since umeemania, we spent time to hardened
```
Comment (toteki) [inline RELEASE_NOTES.md:9]:
```suggestion
v3.0.0 improves upon the _umeemania_ testnet release (v2.0.x) which introduced our **lending** and **oracle** functionality.
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+### Highlights since v1.x
+
+- `x/leverage` module, allowing anyone to:
+  - supply liquidity and earn rewards,
+  - collaterize the supplied liquidity to be able to borrow,
+  - borrow (will charge the user borrow interest),
+  - any umee holder governs the `x/leverage` parameters like: token registry and borrow / supply settings. See more about the parameters in [leverage.proto](https://github.com/umee-network/umee/blob/main/proto/umee/leverage/v1/leverage.proto) file.
```
Comment (toteki) [inline RELEASE_NOTES.md:17]:
```suggestion
- `x/leverage` module, which allows anyone to:
  - supply liquidity (and earn interest)
  - collateralize the supplied assets to enable borrowing
  - borrow (and pay interest)
  - participate in governance of `x/leverage` [parameters](https://github.com/umee-network/umee/blob/main/proto/umee/leverage/v1/leverage.proto) file.
```

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- `x/oracle` module - a decentralized price oracle for the `x/leverage` module, as well as any app built in the Umee blockchain. In the future version we will offer oracle queries through IBC. Any Umee holders will govern oracle parameters defined in [oracle.proto](https://github.com/umee-network/umee/blob/main/proto/umee/oracle/v1/oracle.proto) file.
```
Comment (toteki) [inline RELEASE_NOTES.md:18]:
```suggestion
- `x/oracle` module - a decentralized price oracle for the `x/leverage` module, as well as any app built in the Umee blockchain. UMEE holders set `x/oracle` [parameters](https://github.com/umee-network/umee/blob/main/proto/umee/oracle/v1/oracle.proto) by governance.
```

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+- Cosmos v0.46 upgrade, which features:
+  - [`x/group`](https://tutorials.cosmos.network/tutorials/understanding-group/) module
+  - [`x/nft`](https://github.com/cosmos/cosmos-sdk/tree/v0.46.1/x/nft/spec) module
+  - [Transaction Tips](Transaction Tips and SIGN_MODE_DIRECT_AUX)
+  - [SIGN_MODE_DIRECT_AUX](Transaction Tips and SIGN_MODE_DIRECT_AUX)
```
Comment (toteki) [inline RELEASE_NOTES.md:23]: Note: finish link

File: RELEASE_NOTES.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+Oracle transactions are free only if they are composed from the prevote and vote messages and have gas limit <= 140'000 gas.
+
+#### x/leverage settings
+
+- TODO: important information about queries, and node parameters, eg: https://github.com/umee-network/umee/issues/1400
+
+### Gravity Bridge
+
+In `v1.1.x` (existing mainnet) we disable Gravity Bridge (GB) module due to Ethereum PoS migration (_the merge_).
+This release is the first step to enable GB. We start by enabling validators update end evidence messages (`ValsetConfirm`), but the bridge messages: batch creation, claims (both ways: Ethereum->Cosmos and Cosmos->Ethereum) remain disabled.
```
Comment (toteki) [inline RELEASE_NOTES.md:39]:
```suggestion
In `v1.1.x` (current mainnet) we disabled Gravity Bridge (GB) module due to Ethereum PoS migration (_the merge_).
This release is the first step to re-enable GB. We start by enabling validators update end evidence messages (`ValsetConfirm`), but the bridge messages: batch creation, claims (both ways: Ethereum->Cosmos and Cosmos->Ethereum) remain disabled.
```

File: RELEASE_NOTES.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+### Update notes
+
+Each validator must run:
+
+- Peggo (Gravity Bridge Orchestrator).
+- Price Feeder.
+
+Instructions: https://umeeversity.umee.cc/validators/mainnet-validator.html
+
+Not running above services, or missbehave is slasheable.
```
Comment (toteki) [inline RELEASE_NOTES.md:52]:
```suggestion
Failure to run Peggo and Price Feeder results in being slashed, as do certain types of misbehavior such as submitting incorrect prices.
```

File: RELEASE_NOTES.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+- IBC v5.0
+
+#### Fees
+
+All transactions, except oracle messages, are required to pay gas. We implemented a consensus controlled `protocol_min_gas_price = 0.05uumee`. All **validators must** set their `minimum-gas-prices` settings in `app.yml` to a value at least `0.05uumee` (otherwise the node won't start). Transactions with gas price smaller then `protocol_min_gas_price` will fail during the DeliverTx (transaction execution) phase.
+Oracle transactions are free only if they are composed from the prevote and vote messages and have gas limit <= 140'000 gas.
+
+#### x/leverage settings
+
+- TODO: important information about queries, and node parameters, eg: https://github.com/umee-network/umee/issues/1400
```
Comment (toteki) [inline RELEASE_NOTES.md:34]:
```suggestion
The leverage module is by default compiled without support for the `liquidation_targets` query.

Validators should NOT enable this query on their nodes - it is inefficient due to iterating over all borrower accounts, and can delay time-sensitive consensus operations when a sufficient number of addresses must be checked.

To run a node capable of supporting a liquidator, enable the query at compile time using `LIQUIDATOR=true make install`.
```

File: RELEASE_NOTES.md (hunk 7 above)
Comment (robert-zaremba) [reply RELEASE_NOTES.md]: thanks, done.

File: RELEASE_NOTES.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+Validators should NOT enable this query on their nodes - it is inefficient due to iterating over all borrower accounts, and can delay time-sensitive consensus operations when a sufficient number of addresses must be checked.
+
+To run a node capable of supporting a liquidator, enable the query at compile time using `LIQUIDATOR=true make install`.
+
+### Gravity Bridge
+
+In `v1.1.x` (current mainnet) we disabled Gravity Bridge (GB) module due to Ethereum PoS migration (_the merge_).
+This release is the first step to re-enable GB. We start by enabling validators update end evidence messages (`MsgValsetConfirm` and `MsgValsetUpdatedClaim`), but the bridge messages: batch creation, claims (both ways: Ethereum->Cosmos and Cosmos->Ethereum) remain disabled.
+
+See [Gravity Bridge](https://github.com/umee-network/Gravity-Bridge/blob/umee%2Fv3/module/RELEASE_NOTES.md) Release Notes.
```
Comment (robert-zaremba) [inline RELEASE_NOTES.md:44]:
```suggestion
See [Gravity Bridge](https://github.com/umee-network/Gravity-Bridge/blob/module/v1.5.3-umee-1/module/RELEASE_NOTES.md) Release Notes.
```

File: RELEASE_NOTES.md (hunk 3 above)
Comment (robert-zaremba) [reply RELEASE_NOTES.md:54]:
```suggestion
Instructions: [umeeversity/validator](https://umeeversity.umee.cc/validators/mainnet-validator.html)
```

File: RELEASE_NOTES.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+
+See [Gravity Bridge](https://github.com/umee-network/Gravity-Bridge/blob/module/v1.5.3-umee-1/module/RELEASE_NOTES.md) Release Notes.
+
+### Update notes
+
+Each validator must run:
+
+- Peggo (Gravity Bridge Orchestrator).
+- [Price Feeder](https://github.com/umee-network/umee/tree/main/price-feeder).
+- update `minimum-gas-prices` setting in `app.yml`. See [Fees](#fees) section above.
```
Comment (brandoncomer) [inline RELEASE_NOTES.md]: Fee section doesn't exist. Let's update with the latest of setting `minimum-gas-prices` to 0.
